indexer/blockStack.ts

```diff
@@ -1,4 +1,4 @@
-import { BlockMember } from "../types/indexer";
```

Comment:
No need to modify all the indexer part, this is not related to your PR. You should just implement stellar connection